README.md

```diff
@@ -1 +1,97 @@
-# Hello_Microverse
\ No newline at end of file
+<a name="readme-top"></a>
```

Comment:
Good job but as the previous reviewer requested, remove the boilerplate not the main headers. - Kindly use [Microverse Readme template](https://github.com/microverseinc/readme-template) and make sure all the main sections present in yours and placeholders are replaced with your project and personal data. Only optional section is FAQ. Please remove the template logo on the top, if you have your own logo you can replace it with this one, otherwise please delete it. This is crucial to have a professional and instructive repository. 👌

Reply:
I did my README according to my project. And it's the same as what my coding partners did and it was accepted by reviewers.